README.md

```diff
@@ -46,7 +46,7 @@ YouTube offers a functionality to ban a user, but it does NOT delete previous co
 
 ## 🤔 Pro-Tip If This Seems Sketchy: Limiting The App's Access 🤔
 
-If you feel sketched out about giving the app the required high level permissions to your channel (very understandable), you could instead use the app in 'moderator mode' (set in the config file). First, some context: When you grant access to another channel to be a moderator for your channel, they are able to mark comments for 'held for review', and this permission works through the API as well. 
+If you feel skeptical out about giving the app the required high level permissions to your channel (very understandable), you could instead use the app in 'moderator mode' (set in the config file). First, some context: When you grant access to another channel to be a moderator for your channel, they are able to mark comments for 'held for review', and this permission works through the API as well. 
```

Comment:
> If you feel skeptical out about giving the app[...]

Yeah that's not better

`If you feel skeptical about giving the app [...]`
 would be a better way of saying it

Reply:
Yes, I forgot to take out the "out"